new/mip_process_and_guidelines.md

```diff
@@ -0,0 +1,210 @@
+- Implementations - The implementations must be completed before any MIP is given status “Final”, but it need not be completed before the MIP is merged as draft. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of “rough consensus and running code” is still useful when it comes to resolving many discussions of API details.
+- Copyright Waiver - All MIPs must be in the public domain. See the bottom of this MIP for an example copyright waiver.
+
+## MIP Formats and Templates
+
+MIPs should be written in [markdown] format.
+Image files should be included in a subdirectory of the `assets` folder for that MIP as follow: `assets/mip-X` (for mip **X**). When linking to an image in the MIP, use relative links such as `../assets/mip-X/image.png`.
+
+## MIP Header Preamble
+
+Each MIP must begin with an RFC 822 style header preamble, preceded and followed by three hyphens (`---`). The headers must appear in the following order. Headers marked with "*" are optional and are described below. All other headers are required.
+
+` mip:` <MIP number> (this is determined by the MIP editor)
+
+` title:` <MIP title>
+
+` author:` <a list of the author's or authors' name(s) and/or username(s), or name(s) and email(s). Details are below.>
+
+` * discussions-to:` <url>
+
+` status:` <Draft | Last Call | Accepted | Final | Active | Deferred | Rejected | Superseded>
+
+`* review-period-end: YYYY-MM-DD
+
+` type: `<Standards Track (Core, Networking, Protocol, Schema)  | Informational | Meta>
+
+` * category:` <Core | Networking | Protocol | Schema>
+
+` created:` <date created on, in ISO 8601 (yyyy-mm-dd) format>
+
+` * requires:` <MIP number(s)>
+
+` * replaces:` <MIP number(s)>
+
+` * superseded-by:` <MIP number(s)>
+
+` * resolution:` <url>
+
+#### Author header
+
+The author header optionally lists the names, email addresses or usernames of the authors/owners of the MIP. Those who prefer anonymity may use a username only, or a first name and a username. The format of the author header value must be:
+
+Random J. User &lt;address@dom.ain&gt;
+
+or
+
+Random J. User (@username)
+
+if the email address or GitHub username is included, and
+
+Random J. User
+
+if the email address is not given.
+
+Note: The resolution header is required for Standards Track MIPs only. It contains a URL that should point to an email message or other web resource where the pronouncement about the MIP is made.
+
+While an MIP is a draft, a discussions-to header will indicate the mailing list or URL where the MIP is being discussed. As mentioned above, examples for places to discuss your MIP include [Mercury topics on Gitter](https://gitter.im/libertaria-project/topics), an issue in this repo or in a fork of this repo and [Reddit r/ethereum](https://www.reddit.com/r/libertaria-project/). No discussions-to header is necessary if the MIP is being discussed privately with the author.
```

Comment:
Ah, so it seems the version shown under https://github.com/libertaria-project/mips/pull/1 is actually different and this is likely an updated version here. I'm a bit confused now which rendered page to check for review. Can you please guide me?